pkg/providers/nutanix/config/cp-template.yaml

```
@@ -105,6 +105,16 @@ spec:
           name: awsiamcert
           readOnly: false
 {{- end}}
+{{- if .encryptionProviderConfig }}
+        - hostPath: /etc/kubernetes/enc
+          mountPath: /etc/kubernetes/enc
+          name: encryption-config
+          readOnly: false
```

nit: Mount the encryption config file instead of the whole directory
```suggestion
        - hostPath: /etc/kubernetes/enc/encryption-config.yaml
          mountPath: /etc/kubernetes/enc/encryption-config.yaml
          name: encryption-config
          pathType: File
          readOnly: true
```